hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoTokenHandlerTestBase.java

```diff
@@ -204,8 +211,12 @@ public class CryptoTokenHandlerTestBase {
     @Mock
     protected CryptoSignatureWaiversImpl waivers;
 
+    protected Configuration configuration;
+
     @BeforeEach
     public void setUp() {
+        configuration = new HederaTestConfigBuilder().getOrCreateConfig();
+        tokensConfig = configuration.getConfigData(TokensConfig.class);
```

Comment:
ah, you did it here but missed to remove it on top ;)

Reply:
Ah oaky ! Will remove it ..